Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [InnerTune/App](https://hosted.weblate.org/projects/innertune/app/).


It also includes following components:

* [InnerTune/Fastlane](https://hosted.weblate.org/projects/innertune/fastlane/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/innertune/app/horizontal-auto.svg)

lucasmz-dev commented:
Cute. Weblate just bundled a bunch of commits into one :clown_face: that had nothing to do with each other after I enabled squash commits (per language)

Whatever, let's just not do that in the future IG.
I won't try to fight Git or send a PR with the older commits because then Weblate would have to stay locked a while which isn't great

lucasmz-dev commented:
Since the PR action is still failing, I built some apks (and will routinely do so) in case anyone wants to see how their current work is going!

https://github.com/lucasmz-dev/CONTRIB-InnerTune/releases/

These cannot be considered trustworthy builds. Do not log in, use them only for the purpose of checking translation work.

lucasmz-dev commented:
@z-huang can you close this please? I believe it's causing conflicts.

lucasmz-dev commented:
TODO after merge: Rename Indonesian translations from `id` to `in` as they are currently not used.

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Translated using Weblate (Portuguese)

Currently translated at 100.0% (251 of 251 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (25 of 25 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (251 of 251 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (25 of 25 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (251 of 251 strings)

Co-authored-by: Fernando <frocha2001@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: LucasMZ <git@lucasmz.dev>
Co-authored-by: Sergio Marques <so.boston.android@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt/
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/pt/
Translation: InnerTune/App
Translation: InnerTune/App - Fastlane
Translation: InnerTune/Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Translated using Weblate (French)

Currently translated at 100.0% (25 of 25 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Mickaël Binos <mickaelbinos@outlook.com>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/fr/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Translated using Weblate (Hindi)

Currently translated at 100.0% (25 of 25 strings)

Translated using Weblate (Hindi)

Currently translated at 52.5% (132 of 251 strings)

Translated using Weblate (Hindi)

Currently translated at 31.8% (80 of 251 strings)

Added translation using Weblate (Hindi)

Co-authored-by: Equalizer <harshyadavdos@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hi/
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/hi/
Translation: InnerTune/App
Translation: InnerTune/App - Fastlane
Translation: InnerTune/Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Translated using Weblate (Serbian)

Currently translated at 92.0% (23 of 25 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: LucasMZ <git@lucasmz.dev>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/sr/
Translation: InnerTune/Fastlane

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Translated using Weblate (Korean)

Currently translated at 100.0% (25 of 25 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (251 of 251 strings)

Translated using Weblate (Korean)

Currently translated at 97.6% (245 of 251 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: chah qp <chahqp@gmail.com>
Co-authored-by: justcontributor <kty5663@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ko/
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/ko/
Translation: InnerTune/App
Translation: InnerTune/App - Fastlane
Translation: InnerTune/App - Main

weblate committed:
Update translation files

Updated by "Squash Git commits" add-on in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/
Translation: InnerTune/App - Fastlane

weblate committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 100.0% (25 of 25 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Konrad <weblate.a8joo@simplelogin.com>
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/nb_NO/
Translation: InnerTune/Fastlane

doctorreditshere committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/id/

Commit:
Added translation using Weblate (Tamil)

Commit:
Translated using Weblate (Tamil)

Currently translated at 99.6% (250 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ta/

Commit:
Translated using Weblate (Serbian)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/sr/

Commit:
Translated using Weblate (Arabic)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ar/

Commit:
Added translation using Weblate (Azerbaijani)

MasterRik16 committed:
Translated using Weblate (Italian)

Currently translated at 88.8% (223 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/it/

Commit:
Translated using Weblate (Polish)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pl/

trunars committed:
Translated using Weblate (Bulgarian)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/bg/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (25 of 25 strings)

Translation: InnerTune/App - Fastlane
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/ja/

cyberboh committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/id/

cyberboh committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (25 of 25 strings)

Translation: InnerTune/App - Fastlane
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/id/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (25 of 25 strings)

Translation: InnerTune/App - Fastlane
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/ja/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

SecularSteve committed:
Translated using Weblate (Bosnian)

Currently translated at 100.0% (25 of 25 strings)

Translation: InnerTune/App - Fastlane
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/bs/

SecularSteve committed:
Translated using Weblate (Bosnian)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/bs/

SecularSteve committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (251 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/nl/

thatdamndevan committed:
Translated using Weblate (Malayalam)

Currently translated at 96.0% (24 of 25 strings)

Translation: InnerTune/App - Fastlane
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/ml/

thatdamndevan committed:
Translated using Weblate (Malayalam)

Currently translated at 61.7% (155 of 251 strings)

Translation: InnerTune/App - Main
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ml/

lucasmz-dev commented:
@z-huang hey, know you're busy but, do you think you could do a routine merge for this, just for safety, prevent potential future issues?
Also #1742 afterwards? Just simple translation fixes PRs
